scripts/aviti_run_parameter_parser.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+
+import glob
+import json
+import os
+import sys
+from argparse import ArgumentParser
+from datetime import datetime
+
+from genologics.config import BASEURI, PASSWORD, USERNAME
+from genologics.entities import Process
+from genologics.lims import Lims
+
+DESC = """EPP for parsing run paramters for AVITI runs
+Author: Chuan Wang, Science for Life Laboratory, Stockholm, Sweden
+"""
+
+
+def fetch_fc(process):
+    fc_id = ""
+    if "Load to Flowcell (AVITI)" in process.parent_processes()[0].type.name:
+        fc_id = process.parent_processes()[0].output_containers()[0].name
```

Comment:
Do we want to assign the flowcell ID via the output container or via e.g. a master step field? No strong opinion.

Reply:
Hi, a output container would be a better option because that is the default method for `lims2db` methods.